recipes/apriltag/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "apriltag" %}
+{% set version = "3.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AprilRobotics/apriltag/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 68ef6640f55aac6566ba8d030a4e7d960446d5b9340dfb61e79b9f92a08b1032
+
+build:
+  number: 0
+  script: |
+    cmake -G Ninja -B build -D CMAKE_INSTALL_PREFIX:PATH=$PREFIX -D BUILD_SHARED_LIBS=ON -D BUILD_PYTHON_WRAPPER=OFF
```

Comment:
For Windows, this should be installed to LIBRARY_PREFIX. I would prefer if you used bld.bat/build.sh to write a morw readable script.

Reply:
Do you mean `-D CMAKE_INSTALL_PREFIX:PATH=$LIBRARY_PREFIX`? Where would the executable be installed? And how do I provide different build instructions for different OSes?

When I search for C++ examples, I can find this conda blog https://conda.org/blog/2023-05-18-how-to-use-conda-build/, which also lists the build instructions in the yaml file. Can you give an example of what this should look like?

Reply:
Btw, on Windows I believe this has to be built with `BUILD_SHARED_LIBS=OFF`. Can I enable/disable this based on the OS somehow in the yaml?

Reply:
Please use a `bld.bat` for Windows and `build.sh` for other OSs. Have a look at other C++ recipes in conda-forge to see more usages,